packages/test/test-service-load/testConfig.json

```diff
@@ -12,16 +12,16 @@
             "faultInjectionMaxMs": 150000,
             "optionOverrides":{
                 "odsp":{
-                    "loader":{
-                        "summarizeProtocolTree": [false]
+                    "configurations":{
+                        "Fluid.Driver.Odsp.EnableContainerTypeSummaryUpload": [true],
```

Comment:
can we get rid of this config? at this point we should just always do container upload when we do summarizeProtocolTree

Reply:
we want to avoid config matrix explosion if we don't really need it, so we should aggressively clean things up as they are no longer useful